improver/cli/apply_emos_coefficients.py

```diff
@@ -35,12 +35,21 @@
 
 from improver import cli
 
+# Creates the value_converter that clize needs.
+inputcoeffs = cli.create_constrained_inputcubelist_converter(
+    "emos_coefficient_alpha",
+    lambda cube: cube.name().startswith("emos_coefficient_beta"),
```

Comment:
Under what conditions would a "beta" cube be created that doesn't match this name strictly?  I can't find the relevant line in the cube creation code, which looks like it treats all of the coefficients equivalently in terms of cube name (`"emos_coefficient_{coeff_name}"`).

Reply:
Update: I have now found the situation in which this would be needed with the current code.  However I think there is a better way to structure the coefficients file, suggested below.

Reply:
This has now been modified following altering the structuring of the beta coefficient cube to include a `realization` coordiante. This line is now just "emos_coefficient_beta".